src/Planning/understand-options-tfs-vsts-environments.md

```diff
@@ -1,33 +1,94 @@
+
+While both TFS and VSTS provide the same essential services, VSTS provides your organization these added benefits:
+
+- **Always Latest** - No more expensive and complex upgrade cycles to manage your data center. You always have access to the latest version of the service, with optional access to preview features.
+- **CAPEX -> OPEX** - No servers and infrastructure to support, manage, and upgrade. You can move your organizations from resource optimized model, based on capital expenses (CAPEX), to a market optimised business model, based on operational expenses (OPEX). Read [how Microsoft builds its fast and reliable global network](https://azure.microsoft.com/en-gb/blog/how-microsoft-builds-its-fast-and-reliable-global-network/) to understand how Microsoft is continuously innovating and investing their cloud.
+- **Ease of use** - Easy access from anywhere, with improved connectivity with remote sites, and control when you need it. 
+- **React Fast** - The VSTS live-site management processes are crafted to ensure a deep focus on service health and customer experience, to minimize time to detect, respond to, and mitigate business impacting issues. You have fast access to patches and bug fixes. 
+- **Scalability** - Automatically scales with your needs, with resilience and quality assurance.
+- **Security** - VSTS is hosted entirely in Microsoft Azure datacenters and uses many of the core Azure services including Compute, Storage, Networking, SQL Database, Identity and Access Management Services, and Service Bus. This ensures that VSTS takes advantage of the state of the art capabilities, protection, and industry certifications available from the Azure platform. You get data protection by the experts for data availability, service availability, service security, and data privacy. For details on security, read [Data Protection Overview](https://aka.ms/vsts-security).
+- **Licensing** - Simplified licensing allows you to transition from a capital expenditure (servers and the like) to operational expenditure (subscriptions).
 
 ## Capacity Planning
 
-XXXTBDXXX
+To decide which of the TFS deployment options are viable, you need to consider the advantages of each option, the number of users and projects you will need to support and understand the expected requests per second (RPS) per application tier (AT) server. 
+
+The following table is a quick summary of the three deployment options, with suggested maximum number of conservative users. For a more detailed breakdown of the scenarios and recommended hardware, use the capacity planning workbook from the [TFS Planning, Disaster Avoidance and Recovery](https://vsardata.blob.core.windows.net/projects/Rangers_vsarPlanninglGuide_Everything.zip) guidance.
+
+|Deployment Option|Requests per second (RPS)|Max Users|
+|-----------------|--:|--------:|
+|Single Server|92-180|250-500|
+|Dual Servers (scale up)|476-730|2200-3600|
+|Multi Servers (scale out)|730+|3600+|
+
+>
+> [!TIP]
```

Comment:
Is it worth mentioning that the deployment option can be changed after the fact (or that you can migrate to VSTS)?

Reply:
Yup. Done.